src/main/java/com/atlassian/jira/cloud/jenkins/common/model/PluginSiteData.java

```diff
@@ -0,0 +1,21 @@
+package com.atlassian.jira.cloud.jenkins.common.model;
+
+import java.util.Optional;
+
```

Comment:
Since there can be many sites, created this object so i can loop-de-loop through it when extracting from XML